src/__tests__/__snapshots__/downshift.aria.js.snap

```diff
@@ -0,0 +1,107 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`basic snapshot 1`] = `
+<div
+  aria-expanded="false"
+  aria-haspopup="listbox"
+  aria-labelledby="downshift-0-label"
+  aria-owns="downshift-0-menu"
+  role="combobox"
+>
+  <label
+    data-testid="label"
+    for="downshift-0-input"
+    id="downshift-0-label"
+  >
+    label
+  </label>
+  <input
+    aria-autocomplete="list"
+    aria-controls="downshift-0-menu"
+    aria-expanded="false"
+    aria-labelledby="downshift-0-label"
+    autocomplete="off"
+    data-testid="input"
+    id="downshift-0-input"
+    role="listbox"
+    value="item"
+  />
```

Comment:
The snapshot doesn't cover mid-interaction, i.e. the input has my focus and I have used my arrow keys to "focus" an option. When doing so set `aria-activedescendant` on the input and the value of it should match the ID of the option currently selected or "in focus"

Reply:
Which I see you are doing in the code, so you should be good. Might just be missing setting `aria-selected="true"` on the highlighted item when you're interacting with it, not just when you've set the overall value. (this is for a VoiceOver quirk mainly)

Reply:
We do other stuff for VoiceOver saying the highlighted item. If you want to try this component out with VoiceOver, give this a look: https://deploy-preview-396--downshift.netlify.com/

Reply:
If you want my advice, I'd remove all that stuff and leave it up to the ARIA implementation to do the work. The live region will be fighting with this now better ARIA implementation, and what you might be making up for is poor VoiceOver support, but in JAWS and NVDA it works perfectly and you'll get weird UX side effects for those folks. Arguably it's far more common to use those over VoiceOver anyway.

By correctly setting `aria-activedescendant` with `aria-selected="true"` on the maatching option, you'll get VoiceOver announcing the highlighted option in Chrome easily. Safari is broken, but that's not your fault. Don't make other more common use cases suffer, because of Safari.

I often see implementations calling out the number of available option too, but there really is no need. If my first down arrow press is to open _and_ highlight the first option, the screen reader will tell me how many options there are. It'll say something like "Black, 1 of 6". 

You totally don't need any of that jank.

Reply:
Hmmm... I'll have to test things out myself I guess. This stuff is hard! I wish the standard were more standard. I feel like I'm coding in the IE6 days. Thanks for your feedback!